easybuild/easyconfigs/m/MDSplus-Java/MDSplus-Java-7.131.6-GCCcore-11.3.0-Java-17.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'ConfigureMake'
+
+name = 'MDSplus-Java'
+version = '7.131.6'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://mdsplus.org/'
+description = """MDSplus is a set of software tools for data acquisition
+  and storage and a methodology for management of complex scientific data."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/MDSplus/mdsplus/archive']
+sources = ['stable_release-%s.zip' % version.replace('.', '-')]
+patches = ['%(namelower)s-%(version)s_paths.patch']
+checksums = [
+    'e4762f41e8afb4e22e7fc1ea7c02a51945102dc0a5143e149f8eab0a268bea99',  # stable_release-7-131-6.zip
+    '587afbc3709610614be08c0367716793dd819e71a24608a4821e559bcb5e1051',  # mdsplus-java-7.131.6_paths.patch
+]
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('Autotools', '20220317'),
+    ('Bison', '3.8.2'),
+    ('flex', '2.6.4'),
+    ('gperf', '3.1'),
+]
+
+dependencies = [
+    ('Java', '17', '', True),
```

Comment:
I'm sure it will work with Java/11 since that what we've been using with the 2020b toolchains we've been using for the past couple of years.  If we change the "default" to Java/11 would it be okay to add another EB file with -Java-17 suffixed? And if so, should I also suffix -Java-11 to this one for symmetry?

Reply:
So.. for system level stuff always add a suffix, but for things built under a toolchain we typically don't add suffixes for the "default" version.. is what i would have said it there wasn't like +20 exceptions to that rule already
We also haven't really been to careful about java version that much until just very recently (now basing it on https://whichjdk.com/ recommendations + keeping things simpler for PPC users), so that might explain why it's so sloppy. The suffix doesn't really harm anything and a ton of other easyconfigs do it already, so, sure, go with it.

The only reason to not provide a Java-17 variant is that it building it does provide a small foot-gun, tempting other PRs to to diverge from the default and introduce an avoidable conflict. I wouldn't wanna build that for my users since it just invites them to conflict with all the other modules that use 11
But i won't object to much here if you feel there is value in adding it.